### PR TITLE
feat: [PL-25414]: Hierarchical UI improvements

### PR DESCRIPTION
**Pull request by reetika04**
##### Jira Links:
Screenshot 2022-06-16 at 3 36 11 PM
Screenshot 2022-06-16 at 3 36 18 PM

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by upadhyay-an — DISMISSED** [review]
lgtm

File: src/modules/20-rbac/modals/ResourceScope/views/AccountProjectSelectionRenderer.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      accountIdentifier: accountId,
+      orgIdentifier,
+      identifiers: projects
+    },
+    queryParamStringifyOptions: {
+      arrayFormat: 'repeat'
+    }
+  })
+
+  /* istanbul ignore else */ if (!projects.length) {
```
Comment (avinashmadhwani02) [inline src/modules/20-rbac/modals/ResourceScope/views/AccountProjectSelectionRenderer.tsx:36]: why are we ignoring coverage here?

**Review by avinashmadhwani02 — APPROVED** [review]

File: src/modules/20-rbac/modals/ResourceScope/views/AccountProjectSelectionRenderer.tsx (hunk 1 above)
Comment (reetika04) [reply src/modules/20-rbac/modals/ResourceScope/views/AccountProjectSelectionRenderer.tsx:36]: We are ignoring the else condition as there is no else condition corresponding to this if function.

**Comment by reetika04** [comment]
trigger cypress

**Review by harshilbits — APPROVED** [review]

**Review by avinashmadhwani02 — APPROVED** [review]

**Pull request merged**